bin/webpack-dev-server.js

```diff
@@ -310,8 +314,15 @@ function processOptions(wpOpt) {
 	if(argv["compress"])
 		options.compress = true;
 
-	if(argv["open"] || argv["open"] === "")
-		options.open = argv["open"];
+	if(argv["open"]) {
```

Comment:
do we need the ` || argv["open"] === ""` from the original logical check? (pretty sure we do, but could be wrong)